extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -448,6 +447,7 @@ export class AmpA4A extends AMP.BaseElement {
    * @private
    */
   promiseErrorHandler_(error) {
+    console.log('promiseErrorHandler_', error);
```

Comment:
You'll have to get rid of this for real submission, or convert it to a `user().info()` or something.